server/routes/varyingLicences/handlers/confirmVaryAction.ts

```diff
@@ -12,13 +13,19 @@ export default class ConfirmVaryActionRoutes {
   POST = async (req: Request, res: Response): Promise<void> => {
     const { licenceId } = req.params
     const { answer } = req.body
-    const { user } = res.locals
+    const { user, licence } = res.locals
 
     if (answer === YesOrNo.NO) {
       return res.redirect(`/licence/vary/id/${licenceId}/view-active`)
     }
 
-    const newLicence = await this.licenceService.createVariation(licenceId, user)
+    let newLicence: LicenceSummary
+    const licenceVariations = await this.licenceService.getLicenceVariations(licence.nomsId)
+    if (licenceVariations?.length > 0) {
+      ;[newLicence] = licenceVariations
```

Comment:
the semi-colon is for eslint, the build fails without it. It's because we don't terminate lines with semi-colons so the array at the start of the line could be a continuation of the previous line without the semi-colon. See https://eslint.org/docs/latest/rules/semi

Anyway I've updated to use _.head as you suggested and it's no longer needed :)